api/internal/plugins/loader/loader_test.go

```diff
@@ -78,3 +79,16 @@ func TestLoader(t *testing.T) {
 		}
 	}
 }
+
+func TestLoaderWithWorkingDir(t *testing.T) {
+	p := provider.NewDefaultDepProvider()
+	rmF := resmap.NewFactory(p.GetResourceFactory())
+	fsys := filesys.MakeFsInMemory()
+	c := types.EnabledPluginConfig(types.BploLoadFromFileSys)
+	pLdr := NewLoader(c, rmF, fsys)
+	npLdr := pLdr.LoaderWithWorkingDir("/tmp/dummy")
+	require.Equal(t,
+		"/tmp/dummy",
+		npLdr.Config().FnpLoadingOptions.WorkingDir,
+		"plugin working dir is not set correctly")
```

Comment:
Please add a second assertion that the original loader's working dir did not change